sycl/test/fp64-conv-emu-test-call.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clangxx -O0 -fsycl -fsycl-device-only -fsycl-targets=spir64_gen -fsycl-fp64-conv-emu %s -c -S -emit-llvm -o- | FileCheck %s
+
+// CHECK: define {{.*}} spir_kernel void @_ZTSZ4mainE19fake_kernel_compute(){{.*}} !sycl_used_aspects ![[ASPECTFP64:[0-9]+]]
+// CHECK-NOT: define {{.*}} spir_kernel void @_ZTSZ4mainE19fake_kernel_convert(){{.*}} !sycl_used_aspects
+// CHECK-DAG: ![[ASPECTFP64]] = !{i32 6}
+//
+// Tests if -fsycl-fp64-conv-emu option helps to correctly generate fp64 aspect.
+
+#include "sycl.hpp"
+
+template <typename t, typename Func>
+__attribute__((sycl_kernel)) void kernel_compute(const Func &func) {
+  double a[3] = {1, 2, 3};
+  double b[3] = {1, 2, 3};
+  int i = 1;
+  func(a, b, i);
```

Comment:
Once again, this is a high-level enough test to use SYCL API directly. Same comment applies to the third test

Reply:
Addressed. Thanks for the comment.